runtime/doc/news.txt

```diff
@@ -211,6 +211,9 @@ The following new APIs and features were added.
 • A clipboard provider which uses OSC 52 to copy the selection to the system
   clipboard is now bundled by default. |clipboard-osc52|
 
+• The 'syncout' option buffers screen updates in the terminal emulator
```

Comment:
I'd say "asks the terminal emulator to buffer", otherwise it sounds like it means a terminal emulator as part of nvim itself.

Reply:
in docs I try to use the phrase "host terminal" to mean the terminal that contains the Nvim TUI